src/test/java/ch/uzh/ifi/hase/soprafs22/service/UserServiceTest.java

```diff
@@ -36,14 +55,62 @@ public void setup() {
     testUser.setPassword("1234");
     testUser.setGender(Gender.OTHER);
 
-    // when -> any object is being save in the userRepository -> return the dummy
+    //other user for matching etc
+    otherUser = new User();
+    otherUser.setId(1L);
+    otherUser.setName("otherUser");
+    otherUser.setUsername("testOtherUsername");
+    otherUser.setPassword("1234");
+    otherUser.setGender(Gender.OTHER);
+
+    //give; white card for testing
+    testWhiteCard = new WhiteCard();
+    testWhiteCard.setId(1L);
+    testWhiteCard.setText("CardText");
+    testWhiteCards = new ArrayList<>();
+    testWhiteCards.add(testWhiteCard);
+
+    testGame = new Game();
+    testBlackCard = new BlackCard();
+    testBlackCard.setText("GapText");
+    testBlackCard.setId(1L);
+    otherBlackCard = new BlackCard();
+    otherBlackCard.setId(33L);
+    otherBlackCard.setText("some Text");
+
+    testGame.setId(1L);
+    testGame.setUserId(1L);
+    testGame.setBlackCard(testBlackCard);
+    testGame.setCreationTime(new Date());
```

Comment:
game already has creation date when initializing it

Reply:
@JoMuelle could you please answer as it is your code..

Reply:
fixed